tests/sqllogictests/suites/query/join_small_block_size.test

```diff
@@ -718,5 +718,30 @@ drop table t1;
 statement ok
 drop table t2;
 
+statement ok
+drop table if exists onecolumn;
+
+statement ok
+CREATE TABLE onecolumn (x INT NULL);
+
+statement ok
+INSERT INTO onecolumn(x) VALUES (44), (NULL), (42), (42);
+
+statement ok
+drop table if exists twocolumn;
+
+statement ok
+CREATE TABLE twocolumn (x INT NULL, y INT NULL);
+
+statement ok
+INSERT INTO twocolumn(x, y) VALUES (44,51), (NULL,52), (42,53), (45,45);
+
+query I
+select * from onecolumn as a right anti join twocolumn as b on a.x = b.x and a.x > 42 order by b.x;
```

Comment:
The test in main has same result as the PR.

```
mysql> select * from onecolumn as a right anti join twocolumn as b on a.x = b.x and a.x > 42 order by b.x;
+------+------+
| x    | y    |
+------+------+
|   42 |   53 |
|   45 |   45 |
| NULL |   52 |
```
So is the test  useless for the PR?

Reply:
The max block size here is 1.

Reply:
Btw, the test is right-anti-join (I know the real plan will be left-anti-join due to join order), but the tile is fix left-anti, so it's a little confusing.

You can use `select * from twocolumn as b left anti join onecolumn as a on a.x = b.x and a.x > 42 order by b.x;`  to make it clear.

Reply:
> The max block size here is 1.

oh, got

Reply:
> Btw, the test is right-anti-join (I know the real plan will be left-anti-join due to join order), but the tile is fix left-anti, so it's a little confusing.
> 
> You can use `select * from twocolumn as b left anti join onecolumn as a on a.x = b.x and a.x > 42 order by b.x;` to make it clear.

This test is only for this PR, I will organize them in https://github.com/datafuselabs/databend/pull/13695.